tracee/tracee.go

```diff
@@ -487,3 +495,16 @@ func readArgFromBuff(dataBuff io.Reader) (interface{}, error) {
 	}
 	return res, nil
 }
+
+// DedupStringSlice removes any duplicate items in the incoming slice array as a new slice array
+func DedupStringSlice(in []string) []string {
+	tmpset := make(map[string]bool, len(in))
+	for _, e := range in {
+		tmpset[e] = true
```

Comment:
This logic can be done in one iteration instead of 2:
if _, ok := tmpset[e]; !ok {
    out = append(out, e)
}

Reply:
👍